workflows/prognostic_scream_run/scream_run/run_scream.py

```diff
@@ -0,0 +1,31 @@
+import os
+import sys
+
+# TODO: importlib.resources.files is not available prior to python 3.9
+if sys.version_info.major == 3 and sys.version_info.minor < 9:
+    import importlib_resources  # type: ignore
+elif sys.version_info.major == 3 and sys.version_info.minor >= 9:
+    import importlib.resources as importlib_resources  # type: ignore
+import subprocess
+from .config import ScreamConfig
+from dataclasses import asdict
+
+
+def compose_run_scream_commands(config: ScreamConfig, target_directory: str):
+    os.makedirs(os.path.dirname(target_directory), exist_ok=True)
+    command = ""
+    for key, value in asdict(config).items():
+        if isinstance(value, list):
+            value = ",".join(value)
+        command += f" --{key} {value}"
+    return command
+
+
+def execute_run(command, target_directory: str):
```

Comment:
I think you use `target_directory`, `output_yaml_path`, and `rundir` interchangeably in the code.  It might be good to pick a consistent name.